README.md

```diff
@@ -40,6 +40,39 @@ Some choices for how to run services in your development environment:
     * PostgreSQL 9.5+
     * Mailcatcher 0.7.1
 
+## Setting up Ruby
+
+The easiest way to manage different installations of Ruby is with [rbenv](https://github.com/rbenv/rbenv). After following the installation instructions on the rbenv repo and restarting your terminal, you'll be able to run `rbenv install` in the root directory of the repository. rbenv will handle installing Ruby and switching to it for you when you enter the repo (so no need to worry about your other Ruby installs!) After that, run `gem install bundler` and you'll be all set to run `bundle install`.
```

Comment:
Might be worth adding a note here that people who have `rvm` installed should use rvm instead of installing `rbenv`